client/i18n/locales/english/translations.json

```
@@ -92,6 +92,7 @@
     "big-heading-3": "Earn certifications.",
     "h2-heading": "Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including:",
     "hero-img-description": "freeCodeCamp students at a local study group in South Korea.",
+    "hero-img-alt": "A group of people, including a white man, a black woman, and an Asian woman, gathered around a laptop.",
```

```suggestion
    "hero-img-alt": "A group of people, including a White man, a Black woman, and an Asian woman, gathered around a laptop.",
```

Nit: We should probably capitalise these like we did in the other alt texts.